website/screens/principles/localization/LocalizationPage.tsx

```diff
@@ -193,6 +193,49 @@ const sections = [
           </DxcTable>
         ),
       },
+      {
+        title: "calendar",
+        content: (
+          <DxcTable>
+            <thead>
+              <tr>
+                <th>Label Name</th>
+                <th>Default value</th>
+                <th></th>
+              </tr>
```

Comment:
Last `<th></th>` is not needed. Please remove it from all tables when not needed.

Reply:
I agree with this change but there are multiple components in the same situation, should we apply the change to all the cases in Localization?

Reply:
Yes, it makes no sense to have a header if it is empty and it has no column.